include/ews/ews.hpp

```diff
@@ -12649,6 +12676,16 @@ class attendee final
              << "</t:ResponseType>";
         sstr << "<t:LastResponseTime>" << last_response_time_.to_string()
              << "</t:LastResponseTime>";
+        if (proposed_start_.to_string().empty())
+        {
+            sstr << "<t:ProposedStart>" << proposed_start_.to_string()
+                 << "</t:ProposedStart>";
```

Comment:
Do I see this right that the proposed start is only added to the XML, if it's empty? I would expect it the other way around.

Reply:
Fixed it. Thx.